src/Accounts/Accounts/Environment/AddAzureRMEnvironment.cs

```diff
@@ -216,6 +216,10 @@ public string DataLakeAudience
             HelpMessage = "Specifies URI of the internet resource to fetch environments.")]
         public Uri Uri { get; set; }
 
+        [Parameter(ParameterSetName = EnvironmentPropertiesParameterSet, Position = 23, Mandatory = false, ValueFromPipelineByPropertyName = true,
```

Comment:
```suggestion
        [Parameter(ParameterSetName = EnvironmentPropertiesParameterSet, Mandatory = false, ValueFromPipelineByPropertyName = true,
```
I don't think it's useful to have such a large position number.